src/test/java/com/test/sb/service/CsvFileReaderServiceImplTest.java

```diff
@@ -0,0 +1,60 @@
+package com.test.sb.service;
+
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import static org.junit.Assert.assertEquals;
```

Comment:
let's organize imports the right way here
you can configure the checkstyle plugin to check your tests

Reply:
How can I do it?

Reply:
I've found